blackbox/docs/general/dql/joins.rst

```diff
@@ -230,7 +230,7 @@ available memory, only a certain block size of a relation is loaded at once. The
 whole operation will be repeated with the next block of the first relation once
 scanning the second relation has finished.
 
-This optimisation cannot be applied unless the join is an  **INNER** join and
+This optimization cannot be applied unless the join is an  **INNER** join and
```

Comment:
revert

Reply:
Why revert? Thought we were using American English? Could include this in the 'fixups' PR.

Reply:
reverted

Reply:
yup. for the fixup PR :)